src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx

```diff
@@ -41,10 +41,14 @@ At New Relic, user <DoNotTranslate>**groups**</DoNotTranslate> contain <DoNotTra
 * A specific [role](#roles) (a role being a set of specific, granular [permissions](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions)).
 * One or more accounts.
 
-Organizations with [Pro or Enterprise edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) can have multiple accounts in their organization, and can create custom roles and groups. Our Free and Standard editions allow only a single account in an organization, and don't let you create custom roles or groups.
+Organizations with [Pro or Enterprise edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) can have multiple accounts in their organization, and can [create custom roles and groups](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial/#roles). Our Free and Standard editions allow only a single account in an organization, and don't let you create custom roles or groups.
```

Comment:
pro or enterprise edition should be plural, right? I think the last comma in the last sentence doesn't actually need to be there, but I think that's mostly a preference call.